lib/MusicBrainz/Server/Controller/Account.pm

```diff
@@ -689,12 +689,15 @@ sub donation : Local RequireAuth HiddenOnSlaves
     my $result = $c->model('Editor')->donation_check($c->user);
     $c->detach('/error_500') unless $result;
 
+    # If nag is 0, don't nag - if 1 or -1 then nag
+    my $nag = $result->{nag} != 0;
+
     $c->stash(
         current_view => 'Node',
         component_path => 'account/Donation',
         component_props => {
             days => sprintf("%.0f", $result->{days}),
-            nag => $result->{nag} > 0,
+            nag => $nag,
```

Comment:
Fix looks good, but while you're at it, can you do `boolean_to_json($nag)` here since the type in the file is `+nag: boolean,`?